python/test/unit/runtime/test_subproc.py

```diff
@@ -27,7 +29,7 @@ def kernel_sub(a, b, o, N: tl.constexpr):
 
 def test_compile_in_subproc() -> None:
     config = AttrsDescriptor.from_hints({i: 16 for i in range(4)})
-    multiprocessing.set_start_method('fork')
+    multiprocessing.set_start_method(start_method)
     proc = multiprocessing.Process(target=compile_fn, args=(config, ))
```

Comment:
Pre-existing, but could you update these tests to use contexts so they aren't relying on the global multiprocessing state. i.e. use
```python
mp_ctx = multiprocessing.get_context(start_method)
proc = mp_ctx.Process(target=compile_fn, args=(config, ))

Reply:
done